templates/zerver/emails/notify_new_login.subject

```diff
@@ -1 +1 @@
-A new login to your Zulip account.
+A new login from {{ device_info.device_browser if device_info.device_browser else 'an unknown browser' }} on {{ device_info.device_os if device_info.device_os else 'an unknown operating system' }}
```

Comment:
My thinking is that with Zulip (or Zulip Account Security) as the sender of the email, there was no need to include Zulip again in the subject line.